src/Microsoft.DotNet.Build.Tasks/RunApiCompat.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using ApiCompat;
+
+namespace Microsoft.DotNet.Build.Tasks
+{
+    public class RunApiCompat : Task
+    {
+        [Required]
+        public string Arguments { get; set; }
```

Comment:
I'm not sure that a command line arg string is the best way to pass the parameters. It is probably worth splitting these options into individual properties.